features/support/hooks.rb

```diff
@@ -6,3 +6,13 @@
   Capybara.reset_sessions!
   Capybara.current_driver = :rack_test
 end
+
+
+Before('@poltergeist') do
```

Comment:
I needed to rebase.  (Too many automatic merges on top of automatic merges.)

I'm closing this one and replacing it with #136